### DISCUSSION TITLE
Why I can only see 1 recordings?

### DISCUSSION THREAD

**Discussion by SlavikCA** [roflcoopter/viseron#790] [Q&A]
In UI, Recordings -> {camera} I can only see one, latest recording. 

For example:
![Screenshot 2024-08-06 at 22 57 56](https://github.com/user-attachments/assets/63694893-2069-4e4b-b5ec-c2113a62bb13)

However in the Documentation "Recordings view" - it shows few recordings in the screenshot:
![Docs](https://viseron.netlify.app/img/screenshots/Viseron-screenshot-recordings.png)

Am I missing some settings?

**Comment by SlavikCA** [comment]
I figured!

I need to click that RECORDINGS icon 2nd time.

**Reply by roflcoopter** [reply]
Indeed, not very clear i can admit, sorry for the confusion
